illumos-utils/src/link.rs

```diff
@@ -63,7 +63,7 @@ impl<DL: VnicSource + Clone> VnicAllocator<DL> {
         let name = allocator.next();
         debug_assert!(name.starts_with(VNIC_PREFIX));
         debug_assert!(name.starts_with(VNIC_PREFIX_CONTROL));
-        Dladm::create_vnic(&self.data_link, &name, mac, None)?;
+        Dladm::create_vnic(&self.data_link, &name, mac, None, 9000)?;
```

Comment:
This is not a request for change, just for clarity for my future self or other folks wondering about this 9000 value. The `oxControl<Thing><N>` interfaces are the ones that get handed to things like crucible (`oxControlStorage<N>`) and propolis (`oxControlInstance<N>`). For propolis, we are not talking about the customer ports that go through OPTE, but rather the control interfaces for propolis over which traffic such as API commands and live migration go over. In both of these cases we want to match the physical MTU of 9000.